technical/validations.md

```diff
@@ -7,20 +7,35 @@ these validations will be duplicated in the front-end code for an optimal user e
 ## Entity-level validations
 
 ### Playground
-- The name of the playground is between 3 en 40 characters
-- Two playgrounds cannot be located within .. metres of each other
+- The name of the playground is between 3 en 40 characters ( no whitespaces in the beginning and trailing of the playground name )
+- Two playgrounds cannot be located within .. metres of each other ( Are there any API/Service provided by Netherlands to verify entered/chosen address )
 - The maximum number of playgrounds that can exist within the system is ..
```

Comment:
General comment: fill in the dots (...) please, will be hard to review if the .. are there